apps/desktop/src/components/ProjectSettingsMenuAction.svelte

```diff
@@ -30,6 +30,11 @@
 		openExternalUrl(path);
 	});
 
+	shortcutService.on('open-in-finder', () => {
+		const path = `file://${project.path}`;
+		openExternalUrl(path);
+	});
+
```

Comment:
Should this be leveraging tauri's `revealItemInDir` method with their official `opener` plugin?

https://v2.tauri.app/reference/javascript/opener/#revealitemindir

```suggestion
	shortcutService.on('open-in-finder', async () => {
		try {
			// Use the Opener plugin to reveal the folder in the native file explorer
			await invoke('plugin:opener|reveal_item_in_dir', { path: project.path });
		} catch (e) {
			const message = `
				Failed to reveal directory in file explorer:

				${project.path}
			`;
			showToast({ title: 'Explorer error', message, style: 'error' });

			// Rethrowing for sentry and posthog
			throw e;
		}
	});
```